tests/e2e/configurer/upgrade.go

```diff
@@ -127,6 +127,8 @@ func (uc *UpgradeConfigurer) CreatePreUpgradeState() error {
 
 	chainANode.CreateBalancerPool("pool1A.json", initialization.ValidatorWalletName)
 	chainBNode.CreateBalancerPool("pool1B.json", initialization.ValidatorWalletName)
+	chainANode.CreateStableswapPool("stablePool.json", initialization.ValidatorWalletName)
```

Comment:
Let's also add some logic to execute a swap against one of these pools please

It could look similar to: `TestGeometricTwapMigration`